shub/image/test.py

```diff
@@ -105,4 +109,5 @@ def _run_docker_command(client, image_name, command):
                        stderr=True if statuscode else False,
                        stream=False, timestamps=False)
     utils.debug_log("{} results:\n{}".format(command, logs))
+    client.remove_container(container)
```

Comment:
maybe create a container with `--rm`? if things will go wrong above this line container will not be removed

alternatively make it a context manager with try: finally: